index.js

```diff
@@ -329,6 +347,9 @@ async function fastifyHttpProxy (fastify, opts) {
       }
 
       dest = dest.replace(prefixPathWithVariables, rewritePrefixWithVariables)
+      if (Object.keys(queryParams).length) {
```

Comment:
Separate to other reviews, but can't we only do this?

```suggestion
      if (request.url.includes('?')) {
        dest += querystring.stringify(querystring.parse(request.url.split('?')[1]))
```

Reply:
I changed it to `if (queryParamIndex !== -1) {` since we check [here](https://github.com/fastify/fastify-http-proxy/blob/master/index.js#L319) does that work?

Reply:
Yep, much faster